server/server.js

```diff
@@ -1,15 +1,106 @@
 const express = require("express");
 const app = express();
-const port = process.env.PORT || 5000;
+const bodyParser = require("body-parser");
+const uuid = require("uuid");
+let videos = require("../client/src/exampleresponse.json");
+const port = process.env.PORT || 3000;
+const cors = require("cors");
+const fs = require("fs");
+app.use(express.json());
+app.use(cors());
+app.use(bodyParser.json());
 
 app.listen(port, () => console.log(`Listening on port ${port}`));
 
-// Store and retrieve your videos from here
-// If you want, you can copy "exampleresponse.json" into here to have some data to work with
-let videos = [];
-
 // GET "/"
+// app.get("/", (req, res) => {
+//   res.send({ express: "Your Backend Service is Running" });
+// });
+
+// GET "/"  to get videos
 app.get("/", (req, res) => {
-  // Delete this line after you've confirmed your server is running
-  res.send({ express: "Your Backend Service is Running" });
+  res.send(videos);
+});
+
+app.post("/post", (req, res) => {
+  console.log("Connected to React");
+  res.redirect("/");
+});
+
+// POST "/videos"
+
+app.post("/", (req, res) => {
```

Comment:
Have you checked this function with tooling such as Postman or Thunder Client?
What was your thought process behind putting the validation in a loop?

Passing in the expected data sadly didn't work as expected for me

>   {
    "title": "Louis Test Working 2",
    "url": "https://www.youtube.com/watch?v=X-iSQQgOd1A"
  }